pkg/config/setup/apm.go

```diff
@@ -179,23 +179,20 @@ func setupAPM(config pkgconfigmodel.Setup) {
 		return r
 	})
 
-	config.SetEnvKeyTransformer("apm_config.filter_tags.require", parseKVList("apm_config.filter_tags.require"))
+	config.LoadEnvAsStringSlice("apm_config.filter_tags.require", parseKVList("apm_config.filter_tags.require"))
```

Comment:
I do think it would be worth having some sort of helper function that would handle the error logging. That way we wouldn't have to repeat the key name here, which is only needed for creating an error message.

If this was something like `configsyntax.EnvLoaderAsStringSlice(config, "apm_config.filter_tags.require", configsyntax.ParseKVList)` it would avoid easy to make typos.

Reply:
as discussed, I'll gather all the helper in a dedicated package in another PR.